esmvalcore/config-user.yml

```diff
@@ -1,14 +1,32 @@
 ###############################################################################
-# User's configuration file for the ESMValTool
+# Example user configuration file for ESMValTool
+###############################################################################
+#
+# Note for users:
+# --------------
+# Site-specific entries for different HPC centers are given at the bottom of
+# this file. Comment out/replace as needed.
+#
+###############################################################################
+#
+# Note for developers:
+# -------------------
+# Two identical copies of this file (``ESMValTool/config-user-example.yml`` and
+# ``ESMValCore/esmvalcore/config-user.yml``) exist. If you change one of it,
+# make sure to apply the changes to the other.
+#
 ###############################################################################
 ---
 
 # Directory where results are stored
-output_dir: ./esmvaltool_output
+output_dir: ~/esmvaltool_output
```

Comment:
While I agree that this is a sensible default (it's what I always use in my own configuration file), it is not necessarily the most expected. Most tools write to the current working directory, if you do not tell them where to write.

Reply:
well, we don't, or, we write wherever the config tells it to, if that default may lead to spawning a large number of output dirs, the user will probably blame us first upon finding out they have 300 `esmvaltool_output` dirs because they don't know how to do `pwd` and figure out where they are. Also, they may get lost and say they never ran that particular run :grin:

Reply:
I agree with V here, writing to a path that doesn't depend on the `pwd` seems like the better option here.